### DISCUSSION TITLE
How to disable keybinds in vscode recipe

### DISCUSSION THREAD

**Discussion by kun-codes** [AstroNvim/astrocommunity#1243] [Q&A] [answered]
I want to use the [vscode](https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/recipes/vscode/README.md) recipe in astrocommunity however I don't want [these](https://github.com/AstroNvim/astrocommunity/blob/d5dde44c711859cc2446304de2347230083f43cd/lua/astrocommunity/recipes/vscode/init.lua#L41C1-L113C9) keybinds since I already have keybinds for these in my vscode. How do I override this to remove them?

**Comment by ahmtsen** [comment, marked as answer]
You can add the following snippet in your plugins directory e.g. `plugins/vscode-disable-keymaps.lua` to disable the keymap overrides.

```lua
if not vim.g.vscode then return {} end -- don't do anything in non-vscode instances

return {
  "AstroNvim/astrocore",
  ---@param opts AstroCoreOpts
  opts = function(_, opts) opts.mappings = {} end -- clear mappings,
}
```

**Comment by kun-codes** [comment]
Thanks I tried it for a short period of time and it works. Although I don't know if it would solve the bug where my neovim gets disconnected from vscode after using vscode for sometime. I will update in future.
